.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+# Summary
+
+# Background
+
+# Checklist
```

Comment:
I considered adding a "manually test site works" item, but punted for now. I've got TS and a build step to verify next.js works, and that's good enough for now. I do have a task to add an integration test, which should make this less risky.